packages/application/src/services.ts

```diff
@@ -17,13 +17,13 @@
 // A list of supported docker images for submission
 // TODO: replace this static list with a call to a metadata service
 const dockerImages: { [key: string]: string } = {
-  'tensorflow/tensorflow:1.15.2-py3': 'Tensorflow 1.15.2',
-  'tensorflow/tensorflow:1.15.2-gpu-py3': 'Tensorflow 1.15.2 w/ GPU',
+  'elyra/tensorflow:1.15.2-py3': 'Tensorflow 1.15.2',
+  'elyra/tensorflow:1.15.2-gpu-py3': 'Tensorflow 1.15.2 w/ GPU',
   'tensorflow/tensorflow:2.0.0-py3': 'Tensorflow 2.0',
   'tensorflow/tensorflow:2.0.0-gpu-py3': 'Tensorflow 2.0 w/ GPU',
   'pytorch/pytorch:1.2-cuda10.0-cudnn7-runtime': 'Pytorch 1.2 w/ CUDA-runtime',
   'pytorch/pytorch:1.2-cuda10.0-cudnn7-devel': 'Pytorch 1.2 w/ CUDA-devel',
-  'amancevice/pandas:0.25.0-jupyter': 'Pandas'
+  'amancevice/pandas:1.0.3-jupyter': 'Pandas'
```

Comment:
Not for this PR, but I would recommend we not use the '-jupyter' version of these images.

Reply:
+1 to moving to the non jupyter image

Reply:
The more I think about this the more correct it seems - especially since we're overwriting nearly all the jupyter packages anyway and this would mean less chance of additional conflicts.

@lresende - what do you think?

Reply:
+0, I didn't want to get on the business of maintaining these community images

Reply:
Agreed, but isn't this just a stop-gap to resolve #437?  We should also use non-jupyter-related images as much as possible per my previous comment.

Also, isn't this entire PR a stop-gap to address two current issues with "community images" (lack of `curl` and wrong versions)?  Both of which have issues open to address in kfp-notebook, but I thought there was a higher level of urgency here - thus the stop-gap measures.

Reply:
you are correct, we need to address the user issue with a higher level of urgency.